src/main/java/com/exasol/adapter/sql/SqlFunctionScalar.java

```diff
@@ -46,47 +42,9 @@ public String getFunctionName() {
         return this.function.name();
     }
 
-    public int getNumArgs() {
-        return getArguments().size();
-    }
-
-    public boolean isInfix() {
-        return this.isInfix;
-    }
-
-    public boolean isPrefix() {
-        return this.isPrefix;
-    }
-
     @Override
     public String toSimpleSql() {
-        final List<String> argumentsSql = new ArrayList<>();
-        for (final SqlNode node : this.arguments) {
-            argumentsSql.add(node.toSimpleSql());
-        }
-        if (this.isInfix) {
-            assert argumentsSql.size() == 2;
-            final Map<String, String> functionAliases = new HashMap<>();
-            functionAliases.put("ADD", "+");
-            functionAliases.put("SUB", "-");
-            functionAliases.put("MULT", "*");
-            functionAliases.put("FLOAT_DIV", "/");
-            String realFunctionName = getFunctionName();
-            if (functionAliases.containsKey(getFunctionName())) {
-                realFunctionName = functionAliases.get(getFunctionName());
-            }
-            return "(" + argumentsSql.get(0) + " " + realFunctionName + " " + argumentsSql.get(1) + ")";
-        } else if (this.isPrefix) {
-            assert argumentsSql.size() == 1;
-            final Map<String, String> functionAliases = new HashMap<>();
-            functionAliases.put("NEG", "-");
-            String realFunctionName = getFunctionName();
-            if (functionAliases.containsKey(getFunctionName())) {
-                realFunctionName = functionAliases.get(getFunctionName());
-            }
-            return "(" + realFunctionName + " " + argumentsSql.get(0) + ")";
-        }
-        return getFunctionName() + "(" + String.join(", ", argumentsSql) + ")";
+        return getFunctionName();
```

Comment:
Why are we removing the arguments here?

Reply:
This method is not really used anywhere, we generate the SQL without it at all. But I can't remove it because it comes from the interface